library/axi_sysid/readme.rst

```diff
@@ -0,0 +1,5 @@
+=============
```

Comment:
This information is no information, if we don't want to put more stuff here maybe it would be better just no put any README at all. Less maintenance.  :)

Reply:
I have not yet written any documentation. The file is there just as a reminder. It will contain at least a link to the wiki.